src/components/Layout/Layout.scss

```diff
@@ -293,6 +293,10 @@ body.dark {
     }
 }
 
+main {
+    overflow-x: hidden;
```

Comment:
Can we remove this? It breaks the sticky position of elements on the site. 😅

Reply:
ohh...right....removed!